### ISSUE TITLE
extended metadata within File Metadata Tab doesn't work for editing or deleting

### ISSUE THREAD

**Issue by Lizabrazil** [hydroshare/hydroshare#2733]
If you try to edit or delete a key/value in extended metadata under the **file metadata** tab in a composite resource, it does not work.
![image](https://user-images.githubusercontent.com/13852599/39329384-282f7a5c-496c-11e8-9aae-c63d43795296.png)

**Comment by Lizabrazil** [comment]
@dtarb

**Comment by sblack-usu** [comment]
I looked into this a bit.  It looks like the function that should be called on delete click is never actually bound to the event as the modal window elements do not exist when the binding is called (document ready).  It also looks like the jquery code to get the form id when the delete button is selected won't work.  It's doing a closest search which walk up the parents in the dom.  The form is a sibling of the parent of the button.  It'll need to be searched in a different way.

I made these changes locally and was able to submit the form, but got a 500 error in django so there is some more investigation needed.
